Content.Client/_DV/Options/UI/OptionsTabControlRow.xaml.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Client._DV.Options.UI;
+using Robust.Shared.Configuration;
+
+namespace Content.Client.Options.UI;
+
+public sealed partial class OptionsTabControlRow
+{
+    /// <summary>
+    /// DeltaV: Add a color slider option, backed by a simple string CVar.
```

Comment:
dont need deltav prefix since its moved to _DV

Reply:
So true...